packages/wrangler/src/bundle.ts

```diff
@@ -1,3 +1,4 @@
+import { writeFile } from "fs/promises";
```

Comment:
Let's use the `fs` namespace on line 3. See usage suggestion below. But also if importing from a built-in library then we should use `node:fs/promises`.

```suggestion

```

Reply:
TIL